hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1594,6 +1598,107 @@ public void createVolume(OmVolumeArgs args) throws IOException {
     }
   }
 
+  @Override
+  public VolumeList startCreateVolume(OmVolumeArgs args) throws IOException {
+    try {
+      // TODO: Need to add metrics and Audit log for HA requests
+      if(isAclEnabled) {
+        checkAcls(ResourceType.VOLUME, StoreType.OZONE,
+            ACLType.CREATE, args.getVolume(), null, null);
+      }
+      VolumeList volumeList = volumeManager.createVolume(args);
+      return volumeList;
+    } catch (Exception ex) {
+      throw ex;
```

Comment:
Don't catch and throw same exception. This is an anti-pattern. This needs to be fixed in multiple places in different files.

Reply:
Will update it

Reply:
Done.